subprojects/stdlib/library_api.h

```diff
@@ -1,7 +1,7 @@
 #ifndef __LIBRARY_API_H__
 #define __LIBRARY_API_H__
 
-#include "../src/extern_library_helper.h"
+#include <extern_library_helper.h>
```

Comment:
This doesn't look great on the diff, on the previous commit you added this line.

I think those first two commits are not divided right. Also, the second commit message doesn't make much sense for me.

Move the stdlib part from the first commit to the second, and do the opposite for the modifications on the vinumc. So the first commit would create a base for the creation of  stdlib